components/scripts/lib/config.sh

```diff
@@ -62,6 +62,10 @@ process_arguments() {
   if [ -n "${_arg_interactive+x}" ]; then
     interactive_mode="${_arg_interactive}"
   fi
+
+  if [ -n "${_arg_offline+x}" ]; then
+    offline_mode="${_arg_offline}"
+  fi
```

Comment:
do you know why `debug` is not processed in this method?

Reply:
We can't use a variable named `debug` because we also have a function called `debug`, so I was just using the `_arg_debug` variable directly.

Reply:
What about `debug_mode`? We use the `x_mode` convention in a few places now.

Reply:
Yeah, `debug_mode` would work.

Reply:
We probably want to change that in a different PR though.

Reply:
I've created a separate [issue](https://github.com/gradle/gradle-enterprise-build-validation-scripts/issues/180).